utopia-remix/app/types.ts

```diff
@@ -43,6 +42,20 @@ export function userToCollaborator(user: UserDetails): Collaborator {
   }
 }
 
+export interface AccessRequest {
+  status: AccessRequestStatus
+  token: string
+  user_id: string
+  project_id: string
```

Comment:
I'm assuming this is because you wanted to match the keys from Prisma, but unless there's a good reason to, please don't use snake case here

Reply:
yes, but removed completely.